src/protocols/secure_channel/SimpleSessionResumptionStorage.cpp

```diff
@@ -205,7 +205,7 @@ CHIP_ERROR SimpleSessionResumptionStorage::SaveState(const ScopedNodeId & node,
     ReturnErrorOnFailure(writer.Put(kSharedSecretTag, ByteSpan(sharedSecret.ConstBytes(), sharedSecret.Length())));
 
     CATValues::Serialized cat;
-    peerCATs.Serialize(cat);
+    ReturnErrorOnFailure(peerCATs.Serialize(cat));
```

Comment:
Followup: Why does Serialize return CHIP_ERROR?  It can't fail....  This is just adding code bloat for no reason.

Reply:
Changed function to be `void`